spec/components/previews/flash_component_preview.rb

```diff
@@ -0,0 +1,12 @@
+class FlashComponentPreview < ViewComponent::Preview
```

Comment:
This one is currently broken, something to do with the way previews access helpers. I believe there are workarounds